src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
But one thing... If you leave the OpenSeadragon function outside of the UMD module and you include the file in a `<script>` tag, then what happens is a little bit strange. You create OpenSeadragon and then the UMD reassigns that to itself. Probably more standard would be to put that OpenSeadragon function inside of the UMD module, I think this would be the standard way of doing this.